robot-server/Makefile

```
@@ -70,7 +70,7 @@ clean:
 teardown:
 	$(pipenv) --rm
 
-dist/robotserver-%-py2.py3-none-any.whl: setup.py $(ot_sources)
+dist/robot_server-%-py2.py3-none-any.whl: setup.py $(ot_sources)
```

Does replacing this with the wheel file variable work?

```suggestion
$(wheel_file): setup.py $(ot_sources)
```